src/Api/HttpApi.php

```diff
@@ -95,6 +95,8 @@ protected function handleErrors(ResponseInterface $response)
                 throw  HttpClientException::forbidden($response);
             case 404:
                 throw HttpClientException::notFound($response);
+            case 409:
+                throw HttpClientException::conflict($response);
```

Comment:
Did you see this error happening? I can't see any reference to the `409 Conflict` status code on the official documentation, and I don't remember seeing it while I've been working with this part APIs for a long time.

In case it's unseen, then I believe doesn't make much sense including it, meaning that we can also undo the update here: `src/Exception/HttpClientException.php`

Reply:
@DavidGarciaCat Yes, it happens:
```
try {
    $listId = 'testListId2';

    $response = $mg->emailValidationV4()
        ->deleteBulkJob($listId);
} catch (Exception $e) {
}
```
```
public function deleteBulkJob(string $listId)
{
    Assert::stringNotEmpty($listId);

    $response = $this->httpDelete(sprintf('/v4/address/validate/bulk/%s', $listId));

    print_r($response);

    return $this->hydrateResponse($response, DeleteBulkJobResponse::class);
}
```
```
root@c33e913ec8cd:/var/www/mailgun-php-client# php index.php 
GuzzleHttp\Psr7\Response Object
(
    [reasonPhrase:GuzzleHttp\Psr7\Response:private] => Conflict
    [statusCode:GuzzleHttp\Psr7\Response:private] => 409
    [headers:GuzzleHttp\Psr7\Response:private] => Array
...
```